include/openssl/x509.h

```diff
@@ -644,10 +645,10 @@ int X509_set_version(X509 *x, long version);
 int X509_set_serialNumber(X509 *x, ASN1_INTEGER *serial);
 ASN1_INTEGER *X509_get_serialNumber(X509 *x);
 const ASN1_INTEGER *X509_get0_serialNumber(const X509 *x);
-int X509_set_issuer_name(X509 *x, X509_NAME *name);
-X509_NAME *X509_get_issuer_name(const X509 *a);
-int X509_set_subject_name(X509 *x, X509_NAME *name);
-X509_NAME *X509_get_subject_name(const X509 *a);
+int X509_set_issuer_name(X509 *x, const X509_NAME *name);
+const X509_NAME *X509_get_issuer_name(const X509 *a);
+int X509_set_subject_name(X509 *x, const X509_NAME *name);
+const X509_NAME *X509_get_subject_name(const X509 *a);
```

Comment:
Not for the return type

Reply:
See my above question regarding v3.0

BTW, IMHO these (and other similar) getters should be renamed from `_get_`to `_get0_`.
Can we do this (adding compatibility aliases)?

Reply:
I've reverted these changes on the return type as well, adding a comment here that the functions should better have `get0_` in their names (rather than `get_`).